wdl-lint/tests/lints/deprecated-placeholder-options-v1.1/source.errors

```diff
@@ -0,0 +1,48 @@
+warning[DeprecatedPlaceholderOption]: use of the deprecated `sep` placeholder option
+   ┌─ tests/lints/deprecated-placeholder-options-v1.1/source.wdl:10:32
+   │
+10 │     String bad_sep_option = "~{sep="," numbers}"
+   │                                ^^^^^^^
```

Comment:
I think the span should probably be just the identifier of the node (e.g. `sep`), rather than the entire node.

We can probably add a `name(&self) -> Ident` method to each of the options to facilitate this.

Reply:
On second thought, given that the remediation for the diagnostic is to remove the entire option, having the highlight be the entire option makes sense to me. 

Resolving this comment.